grails-app/controllers/org/pih/warehouse/api/ProductSupplierApiController.groovy

```diff
@@ -28,4 +30,17 @@ class ProductSupplierApiController {
         productSupplierService.delete(params.id)
         render status: 204
     }
+
+    def create(ProductSupplierDetailsCommand productSupplierCommand) {
```

Comment:
Is the naming `productSupplierCommand` here used on purpose (without `Details`)? If not I think it should match the command name

Reply:
you are right, this is inconsistency from my side, so answering - it was not intentional.